packages/schema-blocks/src/instructions/blocks/Duration.tsx

```diff
@@ -0,0 +1,115 @@
+import { BlockConfiguration } from "@wordpress/blocks";
+import { RichText } from "@wordpress/block-editor";
+import { __, sprintf } from "@wordpress/i18n";
+import { TextControl } from "@wordpress/components";
+import moment from "moment";
+import interpolateComponents from "interpolate-components";
+
+import { ReactElement, createElement, useCallback } from "react";
+import BlockInstruction from "../../core/blocks/BlockInstruction";
+import { RenderEditProps, RenderSaveProps } from "../../core/blocks/BlockDefinition";
+
+/**
+ * Duration instruction.
+ */
+export default class Duration extends BlockInstruction {
+	public options: {
+		/**
+		 * The attribute name the value selected in the select control should be saved as.
+		 */
+		name: string;
+		/**
+		 * If it is required that a value is selected.
+		 */
+		required?: boolean;
+		/**
+		 * An optional extra class name or class names.
+		 */
+		className?: string;
+	};
+
+	/**
+	 * The format to be displayed.
+	 */
+	/* translators: %d will be replaced with the number of minutes. */
+	private format: string = __( "%d minutes", "wordpress-seo" );
+
+	/**
+	 * Renders saving the element.
+	 *
+	 * @param props The props.
+	 *
+	 * @returns {JSX.Element} The element to render.
+	 */
+	save( props: RenderSaveProps ): ReactElement | string {
+		const { name } = this.options;
+
+		const value = props.attributes.value as number || 0;
+
+		return <RichText.Content
+			name={ name }
+			value={ sprintf( this.format, value ) }
+			tagName="p"
+		/>;
```

Comment:
We use a TextControl for the edit method, to keep things consistent I would just do: 

```suggestion
		return <p>{ sprintf( this.format, value ) }</p>;
```
```